packages/cozy-client/src/models/contact.js

```diff
@@ -247,3 +248,21 @@ export const getIndexByFamilyNameGivenNameEmailCozyUrl = contact => {
  * @returns {boolean}
  */
 export const isContact = doc => doc._type === CONTACTS_DOCTYPE
+
+/**
+ * @param {CozyClient} client - Instance of CozyClient
+ * @param {object} contact - A contact
```

Comment:
It would be nice to add a IOCozyContact type :pray: https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/types.js